File: PRAK502_2210817110001_AkhmadRaihanRidha.py
def hitung(Nilai1,Nilai2):
    return Nilai1-Nilai2
def mutlak(angka):
    return abs(angka)

a,c,b,d =input().split()
a=int(a)
c=int(c)
b=int(b)
d=int(d)
Hasil=hitung(a,b) + hitung(c,d)
print(mutlak(Hasil))

File: PRAK502_2210817110001_AkhmadRaihanRidha.c
#include <stdio.h>
#include <math.h>
int hitung(int nilai1, int nilai2){
   return nilai1 - nilai2;
  
}
int mutlak(int angka){
   return abs(angka);
}
int main(void)
{ 
 int a,b,x,y,Hasil;
 scanf("%i",&a);
 scanf("%i",&x);
 scanf("%i",&b);
 scanf("%i",&y);
 Hasil = hitung(a,b) + hitung(x,y);
 printf("%i",mutlak(Hasil));
 return 0;
}